homeassistant/components/rachio/switch.py

```diff
@@ -246,9 +281,9 @@ class RachioRainDelay(RachioSwitch):
     _attr_has_entity_name = True
     _attr_translation_key = "rain_delay"
 
-    def __init__(self, controller):
+    def __init__(self, controller) -> None:
         """Set up a Rachio rain delay switch."""
-        self._cancel_update = None
+        self._cancel_update: Union[Callable, None] = None
```

Comment:
```suggestion
        self._cancel_update: Callable | None = None
```
but it would be good to add typing on the Callable as well